spec.bs

```diff
@@ -1350,6 +1438,13 @@ Confirmation, the [=Relying Party=] MUST proceed as follows:
         * Verify that the value of |C|["{{CollectedClientPaymentData/payment}}"]["{{CollectedClientAdditionalPaymentData/payeeOrigin}}"]
             matches the origin of the payee that should have been displayed to the user, if any.
 
+        * Verify that the value of |C|["{{CollectedClientPaymentData/payment}}"]["{{CollectedClientAdditionalPaymentData/facilitatingEntityLogos}}"]
+            is a strict and ordered subset of the logos that should have been displayed to the user, if any.
+
+            Note: The user agent is permitted to not display all logos, but must
+            not include logos in the {{CollectedClientAdditionalPaymentData}}
+            that were not shown to the user.
```

Comment:
```suggestion
           The user agent is not required to display all logos. The user agent must
            not include logos in the {{CollectedClientAdditionalPaymentData}}
            that were not shown to the user.
```

Reply:
Because there is a normative requirement here ("must  not") this should not be a Note.

Reply:
Good catch! Even when not a note this might still not be ok, because this section is not really part of the user agent's specification - it's the "SPC Relying Party Operations".

Possibly we should drop this entirely, or keep it as a note but reword to not have `must` and make it informative only?

Reply:
Does CollectedClientAdditionalPaymentData include these facilitated entity logos? It looks like it only includes *instrument* information. (And the WebIDL suggest this is not a list.)

So how do facilitated entity logos make it into the signed data? If that happens somewhere, then whereever that is specified, it should be a requirement to only include displayed logos.

Then the SPC RP Operations can include a note that the signed data will only include the logos actually displayed.

Reply:
> Does CollectedClientAdditionalPaymentData include these facilitated entity logos? It looks like it only includes instrument information. (And the WebIDL suggest this is not a list.)

It does include them - that is added in this change too :). See https://github.com/w3c/secure-payment-confirmation/pull/294/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR1217 (or step 2.2.2 of https://pr-preview.s3.amazonaws.com/w3c/secure-payment-confirmation/pull/294.html#client-extension-processing-authentication)

Reply:
(Resolving this, please re-open if you are still concerned!)